### PR TITLE
(netflix) Clean up legacy Fast Property code.

### PR DESCRIPTION
**Pull request by zanthrash**
- Removes all of the fast property code.
 - Removes last netflix reliance on V1 of the modal wizard

 - Add the new scope picker to the Persisted Property config stage

@anotherchrisberry @icfantv PTAL

### CODE REVIEW

File: app/scripts/modules/netflix/pipeline/stage/properties/index.js [hunk 1]
```diff
@@ -15,5 +15,5 @@ module.exports = angular.module('spinnaker.netflix.pipeline.stage.property', [
   require('core/deploymentStrategy/deploymentStrategy.module.js'),
   ACCOUNT_SERVICE,
   NAMING_SERVICE,
-  require('../../../fastProperties/modal/wizard/fastPropertyConstraintsSelector.directive'),
+  require('../../../fastProperties/wizard/propertyDetails/fastPropertyConstraintsSelector.directive.js'),
```
Comment (anotherchrisberry) [inline app/scripts/modules/netflix/pipeline/stage/properties/index.js]: probably omit the `.js`

File: app/scripts/modules/netflix/pipeline/stage/properties/create/propertyStage.js [hunk 2]
```diff
@@ -124,14 +99,14 @@ module.exports = angular.module('spinnaker.netflix.pipeline.stage.propertyStage'
...
     };
 
 
 
     vm.applicationsLoaded = false;
-    bindChangeFunctions();
+    // bindChangeFunctions();
     getPropertiesForApp($scope.application.name);
-    vm.applicationSelected(vm.stage.scope.appIdList);
+    // vm.applicationSelected(vm.stage.scope.appIdList);
```
Comment (anotherchrisberry) [inline app/scripts/modules/netflix/pipeline/stage/properties/create/propertyStage.js]: remove commented lines?

File: app/scripts/modules/netflix/fastProperties/view/fastPropertyDetails.controller.js [hunk 3]
```diff
@@ -2,14 +2,14 @@
...
-import { DELETE_FAST_PROPERTY_WIZARD_CONTROLLER } from './wizard/deleteFastPropertyWizard.controller';
+import { UPDATE_FAST_PROPERTY_WIZARD_CONTROLLER } from '../wizard/updateFastPropertyWizard.controller';
+import { DELETE_FAST_PROPERTY_WIZARD_CONTROLLER } from '../wizard/deleteFastPropertyWizard.controller';
 
 module.exports = angular
   .module('spinnaker.netflix.globalFastProperties.details.controller', [
     require('angular-ui-router'),
-    require('./fastProperty.read.service'),
-    require('./fastProperty.write.service'),
+    require('./../fastProperty.read.service.js'),
```
Comment (icfantv) [inline app/scripts/modules/netflix/fastProperties/view/fastPropertyDetails.controller.js]: I don't think the `./` is right.

File: app/scripts/modules/netflix/fastProperties/view/fastPropertyDetails.controller.js [hunk 4]
```diff
@@ -2,14 +2,14 @@
...
+    require('./../fastProperty.write.service.js'),
```
Comment (icfantv) [inline app/scripts/modules/netflix/fastProperties/view/fastPropertyDetails.controller.js]: ditto

**Pull request merged**